src/components/blocks/AccordionBlock.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import type { Template } from 'tinacms'
+import { Accordion } from '@mantine/core'
+import { PageBlocksAccordionblock } from '~tina/__generated__/types'
+import { Icon as Iconify } from '@iconify/react'
+
+export const AccordionBlock = ({ data }: { data: PageBlocksAccordionblock }) => {
+	const [activeIndexes, setActiveIndexes] = React.useState<number[]>([])
+
+	const toggleIndex = (index: number) => {
+		setActiveIndexes((prev) => (prev.includes(index) ? prev.filter((i) => i !== index) : [...prev, index]))
+	}
+
+	return (
+		<div className='grid grid-cols-8 col-span-4 py-8 bg-white'>
+			<div className='col-start-3 col-span-4'>
+				<h1 className='text-[#00D56C] text-3xl font-bold'>{data.blockTitle}</h1>
+				<Accordion multiple>
+					{data.content?.map((item, index) => {
+						return item ? (
+							<Accordion.Item key={index} value={item.accordionTitle || ''}>
+								<Accordion.Control
+									icon={
```

Comment:
This looks great. 

_Note for the future_: Was just wondering if we would be able to add a `transition-all` tailwind class in the future to make the change from '+' to '-' a smoother and maybe some more bottom margin between the header and blockTitle and the first accordion. Regardless, I think it's good to merge